youtube_dl/downloader/external.py

```diff
@@ -191,6 +191,9 @@ def available(cls):
 
     def _call_downloader(self, tmpfilename, info_dict):
         url = info_dict['url']
+
+        is_test = self.params.get('test', False)
```

Comment:
hls.py only used it once as well, so I concluded this was preferred style. Sorry.